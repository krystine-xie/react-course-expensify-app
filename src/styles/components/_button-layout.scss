.button-layout {
    background-color: $grey-blue;
    border: none;
    color: whitesmoke;
    display: inline-block;
    font-size: $font-size-large;
    font-weight: 300;
    line-height: 1;
    padding: $s-size;
    text-align: center;
    text-decoration: none;
}

.button--link {
    background: none; 
    border: none;
    color: whitesmoke; 
    font-size: $font-size-large;
}

.button--secondary {
    background-color: #888888;
}