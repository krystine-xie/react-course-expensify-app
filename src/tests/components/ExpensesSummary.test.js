import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary'; 

// should render summary with one item 
test('should render summary with one item', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={234} />);
    expect(wrapper).toMatchSnapshot();
})

// should render summary with multiple items
test('should render summary with multiple items', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={12} expensesTotal={12345} />);
    expect(wrapper).toMatchSnapshot();
}) 