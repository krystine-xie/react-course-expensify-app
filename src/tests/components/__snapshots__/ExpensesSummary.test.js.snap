// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render summary with multiple items 1`] = `
<div>
  <h1>
    Viewing 
    12
     
    expenses
     totalling 
    $123.45
  </h1>
</div>
`;

exports[`should render summary with one item 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.34
  </h1>
</div>
`;
